13/alpine3.11/Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN addgroup -g 1000 node \
       && case "${alpineArch##*-}" in \
         x86_64) \
           ARCH='x64' \
-          CHECKSUM="3325ec0bff602d30222e0c6c6af81aefa04d7d20946bac65d55dbb09d7d7e1e0" \
+          CHECKSUM="" \
```

Comment:
Will need to re-run one the unofficial builds are ready

Reply:
It's not ready yet, this is a WIP PR ;)